app/views/layouts/application.html.erb

```diff
@@ -20,5 +20,5 @@
     <div class="container">
       <%= yield %>
     </div>
-  </body>
+  </body>  
 </html>
```

Comment:
file nay lam gi the nay

Reply:
file này em giữ lại cái layout của web ấy anh, hôm trước anh cmt là sao lại xóa hẳn đi ấy nên nay em back lại về ban đầu